### ISSUE TITLE
Finding images not working over TLS

### ISSUE THREAD

**Issue by n0t4u** [DefectDojo/django-DefectDojo#4114]
**Bug description**
Hi, I´m new with DefectDojo and I´m testing it to implement it my company. I downloaded and installed DefectDojo using Docker and I´m having problems to view the uploaded images in the findings. 
Digging a little bit I have found that these images are being uploaded to the docker container named django-defectdojo_uwsgi_1 to the path /app/media/uploaded_files, but when I try to see them I get a 404 error.
Looking into the logs I have found that DefectDojo is trying to get the image from the docker container named django-defectdojo_nginx_1 in the path /usr/share/nginx/html/media/uploaded_files/ (see logs below).
I have manually copied the uploaded files from the first container to the second to check this problem and the images can now be seen.
I don´t know if I´m doing something wrong, but it seems weird that the application is uploading the files into one path and then looking for them in another.

PS. I´m using the default configuration.

**Steps to reproduce**
1. Install defectdojo as indicated in the github page.
2. Create a new product type, product, engagement and finding.
3. Access to the finding and upload an image through the Manage Files option.
4. Click on the image uploaded and get a 404 error.

**Expected behavior**
I expect to see the images I have upload in the finding

**Deployment method** *(select with an `X`)*
- [ ] Kubernetes
- [X] Docker
- [ ] setup.bash / legacy-setup.bash

**Environment information**
 - Operating System: Debian 5.9.1-1kali2 (2020-10-29) x86_64 GNU/Linux
 - DefectDojo Commit Message: [2021-03-22 19:11:13 +0100] ec47390e: Bump lxml from 4.6.2 to 4.6.3 (#4108) [ (HEAD -> master, origin/master, origin/HEAD)]

**Screenshots**
Finding with two files uploaded, the first one is not found, the second one is found because I copied it to /usr/share/nginx/html/media/uploaded_files/ path.
![imagen](https://user-images.githubusercontent.com/71253024/112281498-669cf200-8c86-11eb-9601-54cedf345794.png)

404 not found image
![imagen](https://user-images.githubusercontent.com/71253024/112281359-410fe880-8c86-11eb-99e4-c81b3ce2d495.png)

**Console logs**
celeryworker_1  | [23/Mar/2021 12:41:17] INFO [celery.worker.strategy:157] Received task: dojo.tasks.async_dupe_delete[4a4b1a6c-bf58-4b82-9aa1-34b42e93ca80]  
celeryworker_1  | [23/Mar/2021 12:41:17] INFO [celery.app.trace:125] Task dojo.tasks.async_dupe_delete[4a4b1a6c-bf58-4b82-9aa1-34b42e93ca80] succeeded in 0.0038983169999937672s: None
nginx_1         | 2021/03/23 12:41:22 [error] 6#6: *54 open() "/usr/share/nginx/html/media/uploaded_files/5f06dc68-9a7f-42fa-81f1-561d41767dd0.PNG" failed (2: No such file or directory), client: 172.18.0.1, server: , request: "GET /media/uploaded_files/5f06dc68-9a7f-42fa-81f1-561d41767dd0.PNG HTTP/1.1", host: "localhost:8080", referrer: "http://localhost:8080/finding/1"
nginx_1         | 172.18.0.1 - - [23/Mar/2021:12:41:22 +0000] "GET /media/uploaded_files/5f06dc68-9a7f-42fa-81f1-561d41767dd0.PNG HTTP/1.1" 404 153 "http://localhost:8080/finding/1" "Mozilla/5.0 (X11; Linux x86_64; rv:78.0) Gecko/20100101 Firefox/78.0" "-"
nginx_1         | 172.18.0.1 - - [23/Mar/2021:12:41:23 +0000] "GET /favicon.ico HTTP/1.1" 499 0 "-" "Mozilla/5.0 (X11; Linux x86_64; rv:78.0) Gecko/20100101 Firefox/78.0" "-"
uwsgi_1         | [pid: 18|app: 0|req: 106/110] 172.18.0.1 () {40 vars in 663 bytes} [Tue Mar 23 12:41:23 2021] GET /favicon.ico => generated 26621 bytes in 71 msecs (HTTP/1.1 404) 6 headers in 198 bytes (1 switches on core 0)
uwsgi_1         | [pid: 18|app: 0|req: 107/111] 172.18.0.1 () {44 vars in 740 bytes} [Tue Mar 23 12:41:24 2021] GET /alerts/count => generated 12 bytes in 15 msecs (HTTP/1.1 200) 6 headers in 180 bytes (1 switches on core 1)
nginx_1         | 172.18.0.1 - - [23/Mar/2021:12:41:24 +0000] "GET /alerts/count HTTP/1.1" 200 12 "http://localhost:8080/finding/1" "Mozilla/5.0 (X11; Linux x86_64; rv:78.0) Gecko/20100101 Firefox/78.0" "-"

**Comment by valentijnscholten** [comment]
I believe this is fixed in the `dev` branch. Could you try that?

**Comment by n0t4u** [comment]
@valentijnscholten I confirm you that the problem is solved.

You have fixed it as I did yesterday. Just one thing to take into account if someone have the same problem. In addition to modify docker-compose.yml file with the volumes, I had to modify nginx_TLS.conf file because nginx didn´t have the media path configured.

```
location /static {
alias /usr/share/nginx/html/static;
    }
location /media {
      alias /usr/share/nginx/html/media;
    }
location / {
      include /run/defectdojo/uwsgi_pass;
      include /etc/nginx/wsgi_params;
      uwsgi_read_timeout 1800;

```

**Comment by valentijnscholten** [comment]
Makes sense, could you make a small PR against `dev` to add the media mapping to nginx TLS?

**Comment by n0t4u** [comment]
Sorry for the delay answering.
PR done

**Comment by valentijnscholten** [comment]
@n0t4u could you point us to the PR, can't seem to find it.

**Comment by n0t4u** [comment]
It seems that I didn´t to de PR correctly.
I think is now done.
Check this https://github.com/DefectDojo/django-DefectDojo/pull/4371

**Comment by valentijnscholten** [comment]
fixed in 1.15.0
